pkg_defs/kadi/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - configobj
     - requests
     - tables3_api
-    - django <2.0
+    - django
```

Comment:
Letting `django` float (to 2.2) requires https://github.com/sot/kadi/pull/143 and a re-build of `events.db3`.  I think we can and should go in that direction, but commenting that I believe this won't work out of the box.

Reply:
I thought the modern django was actually a requirement for modern dependencies to work, so I was building kadi out of that branch to start.

Reply:
I'm not sure if any of the rest of our OTS stack depends on django, but who knows.  In any case I'm not arguing against the latest django, just giving a heads-up that it will break kadi hard without a database migration.

Reply:
Actually it's worse than just the database, IIRC there are changes that won't allow kadi to import.